gazelle/js/typescript/tsconfig.go

```diff
@@ -140,21 +159,33 @@ func parseTsConfigJSON(cm *TsConfigMap, root, configDir string, tsconfigContent
 	return &config, nil
 }
 
+// Returns the path from the project base to the active tsconfig.json file
+// This is used to build the path from the project base to the file being imported
+// because gazelle seems to resolve files relative to the project base
+// if the passed path is not absolute.
+// Or an empty string if the path is absolute
+func (c TsConfig) getRelativeExpansionIfLocal(importPath string) string {
```

Comment:
Can you think of another name without the "if local"? That seems like an implementation detail. `expandPath` or something like that?

Reply:
Renamed